Code/GraphMol/ShapeHelpers/Wrap/testShapeHelpers.py

```diff
@@ -53,8 +53,12 @@ def test1Shape(self):
                           '1oir_conf.mol')
     m2 = Chem.MolFromMolFile(fileN2)
 
+    self.assertTrue(feq(rdshp.ShapeTanimotoDist(m, m), 0.0))
```

Comment:
It's not wrong to use `assertTrue`  and `feq` here (there are plenty of other tests that do this), but it would be more idiomatic to use `aassertAlmostEqual`
https://docs.python.org/3/library/unittest.html#unittest.TestCase.assertAlmostEqual

Reply:
I try to clean stuff like this up when I see it so that we're continuously improving the code, even if it's not strictly necessary. :-)